### PR TITLE
Add github settings.yml

### PR DESCRIPTION
**Pull request by seanyoung**
Signed-off-by: Sean Young <sean@mess.org>

### CODE REVIEW

**Review by xermicus — CHANGES_REQUESTED** [review]
This seems to be used by the ["Github App"](https://github.com/apps/settings). What is this Github App used for? 

I also wondered, why would security sensitive settings like that be stored inside the repo, allowing anyone with write permissions to the repo to change it's security. And indeed, they come with the following disclaimer:

```
WARNING: Note that this app inherently escalates anyone with push permissions to the admin role, since they can push config settings to the master branch, which will be synced.
```

At least it [can be mitigated](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners).

In summary, it is not clear to me why we need this, while at the same time it introduces a security risk to our repos?

**Review by LucasSte — APPROVED** [review]

**Review by ryjones — APPROVED** [review]

**Comment by seanyoung** [comment]
> In summary, it is not clear to me why we need this, while at the same time it introduces a security risk to our repos?

See reply here https://discord.com/channels/905194001349627914/905205991073804308/1073621894537953370

@ryjones anything you can add? Thanks

**Comment by xermicus** [comment]
> > In summary, it is not clear to me why we need this, while at the same time it introduces a security risk to our repos?
> 
> See reply here https://discord.com/channels/905194001349627914/905205991073804308/1073621894537953370
> 
> @ryjones anything you can add? Thanks

Ah I see, thanks. Can we add a `CODEOWNERS` then to mitigate the security issue?

**Comment by ryjones** [comment]
> > > In summary, it is not clear to me why we need this, while at the same time it introduces a security risk to our repos?
> > 
> > 
> > See reply here https://discord.com/channels/905194001349627914/905205991073804308/1073621894537953370
> > @ryjones anything you can add? Thanks
> 
> Ah I see, thanks. Can we add a `CODEOWNERS` then to mitigate the security issue?

Absolutely. [Do something like this](https://github.com/hyperledger/fabric-chaincode-java/blob/main/CODEOWNERS)

**Comment by ryjones** [comment]
@xermicus see above - I added a CODEOWNERS

**Comment by xermicus** [comment]
> @xermicus see above - I added a CODEOWNERS

Lovely, thanks!

**Review by xermicus — APPROVED** [review]
